src/Yoti.Auth/YotiConstants.cs

```diff
@@ -26,5 +27,10 @@ internal static class YotiConstants
         public const string PostalAddressAttribute = "postal_address";
         public const string StructuredPostalAddressAttribute = "structured_postal_address";
         public const string DateOfBirthAttribute = "date_of_birth";
+
```

Comment:
I would have moved these constants in their relevant classes Profile and ApplicationProfile.
Just a suggestion!

Reply:
Yes fair enough, now that this has become large it makes sense